Dockerfile

```diff
@@ -3,8 +3,8 @@ RUN apt-get update -y
 RUN apt-get install wget build-essential python3-dev libagg-dev libpotrace-dev pkg-config libgl1 -y
 
 WORKDIR /
-COPY requirements.txt /
-RUN pip install -r requirements.txt
+COPY requirements/dev.txt /
+RUN pip install -r dev.txt
```

Comment:
shouldn't this use prod.txt ?

Reply:
It's a good question, if we use `prod.txt` running `docker compose run --build test` doesn't work because pytest is listed as dev dependency. I thought about fixing this directly with docker, or moving pytest as prod dependency, but I found neither solution to be ideal because in general there's quite some work to be done for what concerns have proper testing for the repo - and I concluded we should address that in specialized task / PR 

If you feel we should address this here, happy to do it :)

Reply:
fair, we probably can have prod / dev dockerfiles, but it can be addressed in a separate pr